src/devices/Qmc5883l/FieldRange.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Field ranges of the magnetic sensor can be selected through the register RNG. The full scale field range is
+    /// determined by the application environments.For magnetic clear environment, low field range such as +/- 2gauss
+    /// can be used.The field range goes hand in hand with the sensitivity of the magnetic sensor. The lowest field range
+    /// has the highest sensitivity, therefore, higher resolution.
```

Comment:
```suggestion
    /// Field ranges of the magnetic sensor can be selected through the register RNG. The full scale field range is
    /// determined by the application environments.For magnetic clear environment, low field range such as +/- 2gauss
    /// can be used.The field range goes hand in hand with the sensitivity of the magnetic sensor. The lowest field range
    /// has the highest sensitivity, therefore, higher resolution.
```

RNG=Random Number Generator? Can you expand that, I do not see relationship? Is the information about any registers useful for most of the users?